vagrant/values.yaml

```diff
@@ -170,6 +171,13 @@ otelcol:
                 record_attributes:
                   - key: _SYSTEMD_UNIT
                     value: snap.microk8s.daemon-kubelite.service
+          filter/include_systemd_snap_kubelite:
+            logs:
+              include:
+                match_type: strict
+                record_attributes:
+                  - key: _SYSTEMD_UNIT
+                    value: snap.microk8s.daemon-kubelite.service
```

Comment:
why do we need that?

Reply:
I see, `vagrant` :D

Reply:
To include kubelite lgos in `logs/kubelet` pipeline :) note: we don't have `kubelet` in vagrant env but `kubelite`